src/replica/bulk_load/test/replica_bulk_loader_test.cpp

```diff
@@ -404,7 +405,6 @@ class replica_bulk_loader_test : public replica_test_base
     file_meta _file_meta;
     bulk_load_metadata _metadata;
 
-    std::string APP_NAME = "replica";
```

Comment:
I think this `APP_NAME` is the name of a table rather than the service app (meta/replica, i.e. `replication_options::kReplicaAppType`).  To clear this, we could use some name like `"replica_bulk_loader_test"` as the table name instead of `"replica"`.